### PR TITLE
Fix-Fix Project Item Color Contrast

### PR DESCRIPTION
**Pull request by androidbulb**
Fix Color Contrast in Project Item view Design

### CODE REVIEW

File: app/src/main/java/com/besome/sketch/ProjectsAdapter.java [hunk 1]
```diff
@@ -229,7 +229,7 @@ public void onBindViewHolder(@NonNull RecyclerView.ViewHolder viewHolder, int tr
             if (yB.a(projectMap, "custom_icon")) {
                 Uri uri;
                 String iconFolder = wq.e() + File.separator + scId;
-                if (Build.VERSION.SDK_INT >= 24) {
+                if (Build.VERSION.SDK_INT >= 26) {
```
Comment (PranavPurwar) [inline app/src/main/java/com/besome/sketch/ProjectsAdapter.java:232]: uhmm, what?

File: build.gradle [hunk 2]
```diff
@@ -5,7 +5,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:8.2.0"
+        classpath 'com.android.tools.build:gradle:8.2.2'
```
Comment (PranavPurwar) [inline build.gradle:8]: why switch to single quotes in a single place only?

File: app/src/main/res/layout/view_editor.xml [hunk 3]
```diff
@@ -24,13 +24,14 @@
             <com.besome.sketch.editor.view.palette.PaletteWidget
                 android:id="@+id/palette_widget"
                 android:layout_width="match_parent"
-                android:layout_height="match_parent" />
+                android:layout_height="match_parent"
+                android:visibility="visible"/>
```
Comment (PranavPurwar) [inline app/src/main/res/layout/view_editor.xml:28]: you don't need to specify it as `visible`, its default

File: app/src/main/res/layout/view_editor.xml [hunk 4]
```diff
@@ -24,13 +24,14 @@
...
 
             <com.besome.sketch.editor.view.palette.PaletteFavorite
                 android:id="@+id/palette_favorite"
                 android:layout_width="match_parent"
                 android:layout_height="match_parent"
-                android:visibility="gone" />
+                android:visibility="visible" />
```
Comment (PranavPurwar) [inline app/src/main/res/layout/view_editor.xml:34]: same here

**Comment by remaker17** [comment]
Can you provide final result (screenshot)?

**Pull request closed without merging**